server/src/test/java/org/spine3/server/aggregate/AggregatePartShould.java

```diff
@@ -89,6 +90,11 @@ public void obtain_aggregate_part_constructor() {
         assertNotNull(constructor);
     }
 
+    @Test
+    public void have_TypeInfo() {
+        assertHasPrivateParameterlessCtor(AggregatePart.TypeInfo.class);
```

Comment:
It feels there is a mismatch between the test case name and its content. Why don't we split this into a private ctor test and call to `TypeInfo` test, that will make sure it's present?